include/circt-c/Firtool/Firtool.h

```diff
@@ -217,6 +217,9 @@ MLIR_CAPI_EXPORTED void
 circtFirtoolOptionsSetStripDebugInfo(CirctFirtoolFirtoolOptions options,
                                      bool value);
 
+MLIR_CAPI_EXPORTED void circtFirtoolOptionsSetDisableBlackBoxReourceFile(
+    CirctFirtoolFirtoolOptions options, bool value);
```

Comment:
This function also needs a definition in `lib/CAPI/Firtool/Firtool.cpp`. Other than this looks good to me.